client/state/jetpack-connect/actions.js

```diff
@@ -137,7 +182,8 @@ export default {
 	},
 	createAccount( userData ) {
 		return ( dispatch ) => {
-			tracks.recordEvent( 'jetpack_connect_create_account' );
+			tracksEvent( dispatch, 'jetpack_connect_create_account', {} );
```

Comment:
we are adding the 'calypso_' prefix in the tracksEvent method: https://github.com/Automattic/wp-calypso/pull/5392/files/e2027d7d884d54d8dfc05c66d397d01a01a9b2aa#diff-4082b3f22f0a998cdc936b484d8ea8f9R47

But it's confusing and it is just like that because I was lazy... I'm going to change it to have the proper event names everywhere